File: backend/internal/entities/participant.go
package entities

import "time"

type Participant struct {
	Id         int       `json:"id"`
	Name       string    `json:"name"`
	TeamName   string `json:"team_name" db:"team_name"`
	TeamID     int       `json:"team_id" db:"team_id"`
	Role       string    `json:"role"`
	BirthDate  time.Time `json:"birthdate"`
	BirthPlace string    `json:"birthplace"`
}

File: frontend/src/types/participant.ts
export type ParticipantRequest = {
    name: string;
    team_id: number;
    role: string;
    birthdate: Date;
    birthplace: string;
    user_id: number;
};

export type Participant = {
    id: number;
    name: string;
    team_id: number;
    team_name: string;
    role: string;
    birthdate: string;
    birthplace: string;
    user_id: number;
};
